app/services/food_store.py

```
@@ -1,29 +1,363 @@
+            # Reset counter after reporting
+            _MISSING_FOOD_COUNTER.clear()
+
+
+def reset_missing_food_counter() -> None:
+    """
+    Reset the missing food counter (useful for testing or manual resets).
+
+    RU: Сброс счётчика отсутствующих продуктов (полезно для тестов).
+    EN: Reset the missing food counter (useful for testing or manual resets).
+    """
+    global _MISSING_FOOD_COUNTER
```

Flake8 (7.3.0)

[error] `global _MISSING_FOOD_COUNTER` is unused: name is never assigned in scope

(F824)